### PR TITLE
Made many tweaks and fixes according to tasks set by Josh

### PR DESCRIPTION
**Pull request by hammerfunk**
Updated all the Award Forms, fixed a lot of the copy, changed some JS functionality like the removing of added fields and added auto fill the years for financial.

### CODE REVIEW

File: app/assets/javascripts/frontend/text-character-count.js [hunk 1]
```diff
@@ -1,4 +1,11 @@
 $.fn.charcount = function() {
+  // Adds class to parent so that we can position the question and input closer
+  var prev_el = this.closest(".question-group").prev();
```
Comment (matthewford) [inline app/assets/javascripts/frontend/text-character-count.js]: Identifier 'prev_el' is not in camel case.

File: app/assets/javascripts/frontend/text-character-count.js [hunk 2]
```diff
@@ -1,4 +1,11 @@
...
+  prev_el.addClass("char-spacing");
```
Comment (matthewford) [inline app/assets/javascripts/frontend/text-character-count.js]: Identifier 'prev_el' is not in camel case.

File: app/assets/javascripts/frontend/text-character-count.js [hunk 3]
```diff
@@ -1,4 +1,11 @@
...
+  if (prev_el.hasClass("errors-container")) {
```
Comment (matthewford) [inline app/assets/javascripts/frontend/text-character-count.js]: Identifier 'prev_el' is not in camel case.

File: app/assets/javascripts/frontend/text-character-count.js [hunk 4]
```diff
@@ -1,4 +1,11 @@
...
+    prev_el.prev().append("<span class='char-space'></div>");
```
Comment (matthewford) [inline app/assets/javascripts/frontend/text-character-count.js]: Identifier 'prev_el' is not in camel case.

File: app/assets/stylesheets/frontend/forms.scss [hunk 5]
```diff
@@ -560,9 +560,21 @@ input[type="file"] {
   margin-bottom: -1.8em;
 }
 
+.char-space {
+  display: inline-block;
+  width: 160px;
+  height: 1em;
+}
+
+.errors-container.char-spacing {
```
Comment (matthewford) [inline app/assets/stylesheets/frontend/forms.scss]: Merge rule `.errors-container.char-spacing` with rule on line 27

File: app/assets/stylesheets/frontend/views/award_form.scss [hunk 6]
```diff
@@ -21,6 +21,32 @@
   }
 }
 
+// Show/hide the error step after submitting the form with errors
+.step-error {
+  display: none;
+
+  .help-notice {
+    color: $error-colour;
```
Comment (matthewford) [inline app/assets/stylesheets/frontend/views/award_form.scss]: Properties should be ordered background-image, color

File: app/controllers/accounts_controller.rb [hunk 7]
```diff
@@ -95,7 +95,7 @@ def company_details_params
   end
 
   def contact_settings_params
-    params.require(:user).permit(:prefered_method_of_contact, :subscribed_to_emails, :qae_info_source_other, { qae_info_source: [] })
+    params.require(:user).permit(:prefered_method_of_contact, :subscribed_to_emails, :agree_being_contacted_by_department_of_business, :qae_info_source_other, { qae_info_source: [] })
```
Comment (matthewford) [inline app/controllers/accounts_controller.rb]: Line is too long. [183/100]<br>Redundant curly braces around a hash parameter.

File: app/controllers/application_controller.rb [hunk 8]
```diff
@@ -52,7 +52,7 @@ def configure_permitted_parameters
                :company_name, :company_address_first, :company_address_second,
                :company_city, :company_country, :company_postcode,
                :company_phone_number,
-               :prefered_method_of_contact, :subscribed_to_emails)
+               :prefered_method_of_contact, :subscribed_to_emails, :agree_being_contacted_by_department_of_business)
```
Comment (matthewford) [inline app/controllers/application_controller.rb]: Line is too long. [116/100]

File: app/controllers/content_only_controller.rb [hunk 9]
```diff
@@ -24,6 +35,10 @@ class ContentOnlyController < ApplicationController
 
   def dashboard
     @user_award_forms = current_user.account.form_answers.order("award_type")
+    @user_award_forms_trade = @user_award_forms.where("award_type"=>"trade")
```
Comment (matthewford) [inline app/controllers/content_only_controller.rb]: Surrounding space missing for operator '=>'.

File: app/controllers/content_only_controller.rb [hunk 10]
```diff
@@ -24,6 +35,10 @@ class ContentOnlyController < ApplicationController
...
+    @user_award_forms_innovation = @user_award_forms.where("award_type"=>"innovation")
```
Comment (matthewford) [inline app/controllers/content_only_controller.rb]: Surrounding space missing for operator '=>'.

File: app/controllers/content_only_controller.rb [hunk 11]
```diff
@@ -24,6 +35,10 @@ class ContentOnlyController < ApplicationController
...
+    @user_award_forms_development = @user_award_forms.where("award_type"=>"development")
```
Comment (matthewford) [inline app/controllers/content_only_controller.rb]: Surrounding space missing for operator '=>'.

File: app/controllers/content_only_controller.rb [hunk 12]
```diff
@@ -24,6 +35,10 @@ class ContentOnlyController < ApplicationController
...
+    @user_award_forms_promotion = @user_award_forms.where("award_type"=>"promotion")
```
Comment (matthewford) [inline app/controllers/content_only_controller.rb]: Surrounding space missing for operator '=>'.

File: app/forms/qae_2014_forms/enterprise_promotion/enterprise_promotion_step4.rb [hunk 13]
```diff
@@ -6,7 +6,7 @@ def promotion_step4
         header :support_letters_intro, "" do
           context %(
             <p>
-              As nominator you have sole responsibility for sourcing at least 2 letters of support. Nominations with less than 2 letters will not be accepted. We recommend asking 6-8 supporters to ensure you meet the minimum requirements, as some people might not reply.
+              As nominator you have sole responsibility for sourcing at least 2 letters of support. Nominations with less than 2 letters will not be accepted. We recommend asking 3-5 supporters to ensure you meet the minimum requirements, as some people might not reply.
```
Comment (matthewford) [inline app/forms/qae_2014_forms/enterprise_promotion/enterprise_promotion_step4.rb]: Line is too long. [270/100]

File: app/forms/qae_2014_forms/enterprise_promotion/enterprise_promotion_step5.rb [hunk 14]
```diff
@@ -2,7 +2,7 @@ class QAE2014Forms
   class << self
     def promotion_step5
       @promotion_step5 ||= proc do
-        upload :innovation_materials, "If there is additional material you feel would help us to assess your entry then you can add up to 4 files or links here." do
+        upload :innovation_materials, "If there is additional material you feel would help us to assess your nomination then you can add up to 4 files or links here." do
```
Comment (matthewford) [inline app/forms/qae_2014_forms/enterprise_promotion/enterprise_promotion_step5.rb]: Line is too long. [169/100]

File: app/forms/qae_2014_forms/innovation/innovation_step4.rb [hunk 15]
```diff
@@ -15,10 +15,23 @@ def innovation_step4
         header :complete_now_header, "" do
           conditional :corp_responsibility_form, :complete_now
           context %(
-            <p>The Declaration of Corporate Responsibility is a chance for you to outline your responsible business conduct, and its social, economic and environmental impact.</p>
-            <p>You don't have to demonstrate strength in all of the areas below. </p>
-            <p>If you have too many iniatives, just outline the ones you think are most relevant/important.</p>
-                    )
+            <p>
+              The Declaration of Corporate Responsibility is a chance for you to outline your responsible business conduct, and its social, economic and environmental impact.
```
Comment (matthewford) [inline app/forms/qae_2014_forms/innovation/innovation_step4.rb]: Line is too long. [174/100]

File: app/forms/qae_2014_forms/innovation/innovation_step4.rb [hunk 16]
```diff
@@ -15,10 +15,23 @@ def innovation_step4
...
+            </p>
+            <p>
+              You don't have to demonstrate strength in all of the areas below.
+            </p>
+            <p>
+              The guidance notes below each section are not exhaustive. Answer the questions in a way that best suits your organisation.
```
Comment (matthewford) [inline app/forms/qae_2014_forms/innovation/innovation_step4.rb]: Line is too long. [136/100]

File: app/forms/qae_2014_forms/innovation/innovation_step4.rb [hunk 17]
```diff
@@ -15,10 +15,23 @@ def innovation_step4
...
+            </p>
+            <p>
+              If you can give quantitative evidence of your initiatives/improvement/success, then do so.
```
Comment (matthewford) [inline app/forms/qae_2014_forms/innovation/innovation_step4.rb]: Line is too long. [104/100]

File: app/forms/qae_2014_forms/innovation/innovation_step4.rb [hunk 18]
```diff
@@ -15,10 +15,23 @@ def innovation_step4
...
+              </p>
+            <p>
+              If you have too many initiatives, just outline the ones you think are most relevant/important.
```
Comment (matthewford) [inline app/forms/qae_2014_forms/innovation/innovation_step4.rb]: Line is too long. [108/100]

File: app/forms/qae_2014_forms/innovation/innovation_step4.rb [hunk 19]
```diff
@@ -85,7 +98,7 @@ def innovation_step4
 
         confirm :declaration_of_corporate_responsibility, "" do
           required
-          text "I declare corporate responsibility lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.<br><br>Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
+          text "I am not aware of any matter which might cast doubt on the worthiness of my organisation to receive a Queen's Award for Enterprise."
```
Comment (matthewford) [inline app/forms/qae_2014_forms/innovation/innovation_step4.rb]: Line is too long. [148/100]

File: app/forms/qae_2014_forms/innovation/innovation_step6.rb [hunk 20]
```diff
@@ -64,7 +64,7 @@ def innovation_step6
 
         submit "Submit application" do
           notice %(
-            <p>If you have answered all the questions, you can submit your application now. You will be able to edit it any time before 23:59 on the last working day of September.</p>
+            <p>If you’ve answered all the questions, you can submit your application now. You will be able to edit it any time before 23:59 on the last working day of September.</p>
```
Comment (matthewford) [inline app/forms/qae_2014_forms/innovation/innovation_step6.rb]: Line is too long. [181/100]

File: app/controllers/accounts_controller.rb [hunk 21]
```diff
@@ -87,18 +87,42 @@ def update_password_settings
...
   end
 
   def contact_settings_params
-    params.require(:user).permit(:prefered_method_of_contact, :subscribed_to_emails, :qae_info_source_other, { qae_info_source: [] })
+    params.require(:user).permit(
+      :prefered_method_of_contact,
+      :subscribed_to_emails,
+      :agree_being_contacted_by_department_of_business,
+      :qae_info_source_other,
+      { qae_info_source: [] }
```
Comment (matthewford) [inline app/controllers/accounts_controller.rb]: Redundant curly braces around a hash parameter.

File: app/controllers/application_controller.rb [hunk 22]
```diff
@@ -43,16 +43,34 @@ def current_account
...
-      u.permit(:email, :current_password, :password, :password_confirmation,
-               :completed_registration,
-               :title, :first_name, :last_name,
-               :job_title, :phone_number,
-               :company_name, :company_address_first, :company_address_second,
-               :company_city, :company_country, :company_postcode,
-               :company_phone_number,
-               :prefered_method_of_contact, :subscribed_to_emails)
+      u.permit(:email,
+        :current_password,
```
Comment (matthewford) [inline app/controllers/application_controller.rb]: Align the parameters of a method call if they span more than one line.

File: app/controllers/application_controller.rb [hunk 23]
```diff
@@ -43,16 +43,34 @@ def current_account
...
+        :password,
```
Comment (matthewford) [inline app/controllers/application_controller.rb]: Align the parameters of a method call if they span more than one line.

File: app/controllers/application_controller.rb [hunk 24]
```diff
@@ -43,16 +43,34 @@ def current_account
...
+        :password_confirmation,
```
Comment (matthewford) [inline app/controllers/application_controller.rb]: Align the parameters of a method call if they span more than one line.

File: app/controllers/application_controller.rb [hunk 25]
```diff
@@ -43,16 +43,34 @@ def current_account
...
+        :completed_registration,
```
Comment (matthewford) [inline app/controllers/application_controller.rb]: Align the parameters of a method call if they span more than one line.

File: app/controllers/application_controller.rb [hunk 26]
```diff
@@ -43,16 +43,34 @@ def current_account
...
+        :title,
```
Comment (matthewford) [inline app/controllers/application_controller.rb]: Align the parameters of a method call if they span more than one line.

File: app/controllers/application_controller.rb [hunk 27]
```diff
@@ -43,16 +43,34 @@ def current_account
...
+        :first_name,
```
Comment (matthewford) [inline app/controllers/application_controller.rb]: Align the parameters of a method call if they span more than one line.

File: app/controllers/application_controller.rb [hunk 28]
```diff
@@ -43,16 +43,34 @@ def current_account
...
+        :last_name,
```
Comment (matthewford) [inline app/controllers/application_controller.rb]: Align the parameters of a method call if they span more than one line.

File: app/controllers/application_controller.rb [hunk 29]
```diff
@@ -43,16 +43,34 @@ def current_account
...
+        :job_title,
```
Comment (matthewford) [inline app/controllers/application_controller.rb]: Align the parameters of a method call if they span more than one line.

File: app/controllers/application_controller.rb [hunk 30]
```diff
@@ -43,16 +43,34 @@ def current_account
...
+        :phone_number,
```
Comment (matthewford) [inline app/controllers/application_controller.rb]: Align the parameters of a method call if they span more than one line.

File: app/controllers/application_controller.rb [hunk 31]
```diff
@@ -43,16 +43,34 @@ def current_account
...
+        :company_name,
```
Comment (matthewford) [inline app/controllers/application_controller.rb]: Align the parameters of a method call if they span more than one line.

File: app/controllers/application_controller.rb [hunk 32]
```diff
@@ -43,16 +43,34 @@ def current_account
...
+        :company_address_first,
```
Comment (matthewford) [inline app/controllers/application_controller.rb]: Align the parameters of a method call if they span more than one line.

File: app/controllers/application_controller.rb [hunk 33]
```diff
@@ -43,16 +43,34 @@ def current_account
...
+        :company_address_second,
```
Comment (matthewford) [inline app/controllers/application_controller.rb]: Align the parameters of a method call if they span more than one line.

File: app/controllers/application_controller.rb [hunk 34]
```diff
@@ -43,16 +43,34 @@ def current_account
...
+        :company_city,
```
Comment (matthewford) [inline app/controllers/application_controller.rb]: Align the parameters of a method call if they span more than one line.

File: app/controllers/application_controller.rb [hunk 35]
```diff
@@ -43,16 +43,34 @@ def current_account
...
+        :company_country,
```
Comment (matthewford) [inline app/controllers/application_controller.rb]: Align the parameters of a method call if they span more than one line.

File: app/controllers/application_controller.rb [hunk 36]
```diff
@@ -43,16 +43,34 @@ def current_account
...
+        :company_postcode,
```
Comment (matthewford) [inline app/controllers/application_controller.rb]: Align the parameters of a method call if they span more than one line.

File: app/controllers/application_controller.rb [hunk 37]
```diff
@@ -43,16 +43,34 @@ def current_account
...
+        :company_phone_number,
```
Comment (matthewford) [inline app/controllers/application_controller.rb]: Align the parameters of a method call if they span more than one line.

File: app/controllers/application_controller.rb [hunk 38]
```diff
@@ -43,16 +43,34 @@ def current_account
...
+        :prefered_method_of_contact,
```
Comment (matthewford) [inline app/controllers/application_controller.rb]: Align the parameters of a method call if they span more than one line.

File: app/controllers/application_controller.rb [hunk 39]
```diff
@@ -43,16 +43,34 @@ def current_account
...
+        :subscribed_to_emails,
```
Comment (matthewford) [inline app/controllers/application_controller.rb]: Align the parameters of a method call if they span more than one line.

File: app/controllers/application_controller.rb [hunk 40]
```diff
@@ -43,16 +43,34 @@ def current_account
...
+        :agree_being_contacted_by_department_of_business
```
Comment (matthewford) [inline app/controllers/application_controller.rb]: Align the parameters of a method call if they span more than one line.

File: app/controllers/application_controller.rb [hunk 41]
```diff
@@ -43,16 +43,35 @@ def current_account
   private
 
   def configure_permitted_parameters
-    devise_parameter_sanitizer.for(:sign_up) { |u| u.permit(:email, :password, :password_confirmation, :agreed_with_privacy_policy) }
+    devise_parameter_sanitizer.for(:sign_up) { |u| u.permit(
+      :email,
+      :password,
+      :password_confirmation,
+      :agreed_with_privacy_policy
+    ) }
```
Comment (matthewford) [inline app/controllers/application_controller.rb]: Expression at 51, 7 should be on its own line.

File: app/controllers/application_controller.rb [hunk 42]
```diff
@@ -43,16 +43,35 @@ def current_account
   private
 
   def configure_permitted_parameters
-    devise_parameter_sanitizer.for(:sign_up) { |u| u.permit(:email, :password, :password_confirmation, :agreed_with_privacy_policy) }
+    devise_parameter_sanitizer.for(:sign_up) { |u| u.permit(
```
Comment (matthewford) [inline app/controllers/application_controller.rb]: Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.

File: app/forms/qae_2014_forms/international_trade/international_trade_step4.rb [hunk 43]
```diff
@@ -14,15 +14,29 @@ def trade_step4
 
         header :complete_now_header, "" do
           context %(
-            <p>The Declaration of Corporate Responsibility is a chance for you to outline your responsible business conduct, and its social, economic and environmental impact.</p>
-            <p>You don't have to demonstrate strength in all of the areas below.</p>
-            <p>If you have too many initiatives, just outline the ones you think are most relevant/important.</p>
-                    )
+            <p>
+              The Declaration of Corporate Responsibility is a chance for you to outline your responsible business conduct, and its social, economic and environmental impact.
```
Comment (matthewford) [inline app/forms/qae_2014_forms/international_trade/international_trade_step4.rb]: Line is too long. [174/100]

File: app/forms/qae_2014_forms/international_trade/international_trade_step4.rb [hunk 44]
```diff
@@ -14,15 +14,29 @@ def trade_step4
...
+            </p>
+            <p>
+              You don't have to demonstrate strength in all of the areas below.
+            </p>
+            <p>
+              The guidance notes below each section are not exhaustive. Answer the questions in a way that best suits your organisation.
```
Comment (matthewford) [inline app/forms/qae_2014_forms/international_trade/international_trade_step4.rb]: Line is too long. [136/100]

File: app/forms/qae_2014_forms/international_trade/international_trade_step4.rb [hunk 45]
```diff
@@ -14,15 +14,29 @@ def trade_step4
...
+            </p>
+            <p>
+              If you can give quantitative evidence of your initiatives/improvement/success, then do so.
```
Comment (matthewford) [inline app/forms/qae_2014_forms/international_trade/international_trade_step4.rb]: Line is too long. [104/100]

File: app/forms/qae_2014_forms/international_trade/international_trade_step4.rb [hunk 46]
```diff
@@ -14,15 +14,29 @@ def trade_step4
...
+              </p>
+            <p>
+              If you have too many initiatives, just outline the ones you think are most relevant/important.
```
Comment (matthewford) [inline app/forms/qae_2014_forms/international_trade/international_trade_step4.rb]: Line is too long. [108/100]

File: app/forms/qae_2014_forms/international_trade/international_trade_step4.rb [hunk 47]
```diff
@@ -83,7 +97,7 @@ def trade_step4
 
         confirm :declaration_of_corporate_responsibility, "" do
           required
-          text "I declare corporate responsibility lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.<br><br>Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
+          text "I am not aware of any matter which might cast doubt on the worthiness of my organisation to receive a Queen's Award for Enterprise."
```
Comment (matthewford) [inline app/forms/qae_2014_forms/international_trade/international_trade_step4.rb]: Line is too long. [148/100]

File: app/forms/qae_2014_forms/international_trade/international_trade_step6.rb [hunk 48]
```diff
@@ -73,7 +73,7 @@ def trade_step6
 
         submit "Submit application" do
           notice %(
-            <p>If you have answered all the questions, you can submit your application now. You will be able to edit it any time before 23:59 on the last working day of September.</p>
+            <p>If you’ve answered all the questions, you can submit your application now. You will be able to edit it any time before 23:59 on the last working day of September.</p>
```
Comment (matthewford) [inline app/forms/qae_2014_forms/international_trade/international_trade_step6.rb]: Line is too long. [181/100]

File: app/forms/qae_2014_forms/sustainable_development/sustainable_development_step1.rb [hunk 49]
```diff
@@ -173,6 +173,16 @@ def development_step1
             </div>
                     )
           conditional :external_are_aware_about_award, "no"
+        end
+
+        header :external_organization_or_individual_info_header_some, "" do
+          context %(
+            <div class="application-notice help-notice">
+              <p>
+                We recommend that you notify all the contributors to your product/service/management approach of this entry.
```
Comment (matthewford) [inline app/forms/qae_2014_forms/sustainable_development/sustainable_development_step1.rb]: Line is too long. [124/100]

File: app/forms/qae_2014_forms/sustainable_development/sustainable_development_step2.rb [hunk 50]
```diff
@@ -44,15 +44,13 @@ def development_step2
           words_max 500
         end
 
-        header :provide_evidence_header, "" do
-          context %(
-            <p>Where possible, provide evidence of your product/service/management approach's contribution to each of the dimensions of sustainable development below. If your contribution is weak in any of them, describe the relevant actions taken to improve this.</p>
-                    )
+        header :provide_evidence_header, "Where possible, provide evidence of your product/service/management approach's contribution to each of the dimensions of sustainable development below. If your contribution is weak in any of them, describe the relevant actions taken to improve this." do
```
Comment (matthewford) [inline app/forms/qae_2014_forms/sustainable_development/sustainable_development_step2.rb]: Line is too long. [295/100]

File: app/forms/qae_2014_forms/sustainable_development/sustainable_development_step3.rb [hunk 51]
```diff
@@ -313,7 +313,7 @@ def development_step3
           words_max 300
         end
 
-        textarea :investments_details, "Please enter details of all investments in your product/service/management approach. *Include all investments made both during and prior to your entry period.* Also include the year(s) in which they were made." do
+        textarea :investments_details, "Please enter details of all investments in your product/service/management approach. <span class='text-underline'>Include all investments made both during and prior to your entry period.</span> Also include the year(s) in which they were made." do
```
Comment (matthewford) [inline app/forms/qae_2014_forms/sustainable_development/sustainable_development_step3.rb]: Line is too long. [287/100]

Comment (hammerfunk) [inline app/forms/qae_2014_forms/sustainable_development/sustainable_development_step3.rb]: Ignore

File: app/forms/qae_2014_forms/sustainable_development/sustainable_development_step2.rb (hunk 50 above)
Comment (hammerfunk) [inline app/forms/qae_2014_forms/sustainable_development/sustainable_development_step2.rb]: Ignore

File: app/forms/qae_2014_forms/sustainable_development/sustainable_development_step1.rb (hunk 49 above)
Comment (hammerfunk) [inline app/forms/qae_2014_forms/sustainable_development/sustainable_development_step1.rb]: Ignore

File: app/forms/qae_2014_forms/international_trade/international_trade_step6.rb (hunk 48 above)
Comment (hammerfunk) [inline app/forms/qae_2014_forms/international_trade/international_trade_step6.rb]: Ignore

File: app/forms/qae_2014_forms/international_trade/international_trade_step4.rb (hunk 47 above)
Comment (hammerfunk) [inline app/forms/qae_2014_forms/international_trade/international_trade_step4.rb]: Ignore

File: app/forms/qae_2014_forms/international_trade/international_trade_step4.rb (hunk 45 above)
Comment (hammerfunk) [inline app/forms/qae_2014_forms/international_trade/international_trade_step4.rb]: Ignore

File: app/forms/qae_2014_forms/international_trade/international_trade_step4.rb (hunk 44 above)
Comment (hammerfunk) [inline app/forms/qae_2014_forms/international_trade/international_trade_step4.rb]: Ignore

File: app/forms/qae_2014_forms/international_trade/international_trade_step4.rb (hunk 43 above)
Comment (hammerfunk) [inline app/forms/qae_2014_forms/international_trade/international_trade_step4.rb]: Ignore

File: app/forms/qae_2014_forms/international_trade/international_trade_step4.rb (hunk 46 above)
Comment (hammerfunk) [inline app/forms/qae_2014_forms/international_trade/international_trade_step4.rb]: Ignore

File: app/forms/qae_2014_forms/international_trade/international_trade_step4.rb (hunk 43 above)
Comment (matthewford) [inline app/forms/qae_2014_forms/international_trade/international_trade_step4.rb]: Line is too long. [174/100]

File: app/forms/qae_2014_forms/international_trade/international_trade_step4.rb (hunk 44 above)
Comment (matthewford) [inline app/forms/qae_2014_forms/international_trade/international_trade_step4.rb]: Line is too long. [136/100]

File: app/forms/qae_2014_forms/international_trade/international_trade_step4.rb (hunk 45 above)
Comment (matthewford) [inline app/forms/qae_2014_forms/international_trade/international_trade_step4.rb]: Line is too long. [104/100]

File: app/forms/qae_2014_forms/international_trade/international_trade_step4.rb (hunk 46 above)
Comment (matthewford) [inline app/forms/qae_2014_forms/international_trade/international_trade_step4.rb]: Line is too long. [108/100]

File: app/forms/qae_2014_forms/international_trade/international_trade_step4.rb (hunk 47 above)
Comment (matthewford) [inline app/forms/qae_2014_forms/international_trade/international_trade_step4.rb]: Line is too long. [148/100]

File: app/forms/qae_2014_forms/international_trade/international_trade_step6.rb (hunk 48 above)
Comment (matthewford) [inline app/forms/qae_2014_forms/international_trade/international_trade_step6.rb]: Line is too long. [181/100]

File: app/forms/qae_2014_forms/sustainable_development/sustainable_development_step1.rb (hunk 49 above)
Comment (matthewford) [inline app/forms/qae_2014_forms/sustainable_development/sustainable_development_step1.rb]: Line is too long. [124/100]

File: app/forms/qae_2014_forms/sustainable_development/sustainable_development_step2.rb (hunk 50 above)
Comment (matthewford) [inline app/forms/qae_2014_forms/sustainable_development/sustainable_development_step2.rb]: Line is too long. [295/100]

File: app/forms/qae_2014_forms/sustainable_development/sustainable_development_step3.rb (hunk 51 above)
Comment (matthewford) [inline app/forms/qae_2014_forms/sustainable_development/sustainable_development_step3.rb]: Line is too long. [287/100]

File: app/forms/qae_2014_forms/sustainable_development/sustainable_development_step4.rb [hunk 52]
```diff
@@ -14,10 +14,26 @@ def development_step4
 
         header :complete_now_header, "" do
           context %(
-            <p>The Declaration of Corporate Responsibility is a chance for you to outline your responsible business conduct, and its social, economic and environmental impact.</p>
-            <p>You don't have to demonstrate strength in all of the areas below.</p>
-            <p>If you have too many initiatives, just outline the ones you think are most relevant/important.</p>
-                    )
+            <p>
+              The Declaration of Corporate Responsibility is a chance for you to outline your responsible business conduct, and its social, economic and environmental impact.
```
Comment (matthewford) [inline app/forms/qae_2014_forms/sustainable_development/sustainable_development_step4.rb]: Line is too long. [174/100]

File: app/forms/qae_2014_forms/sustainable_development/sustainable_development_step4.rb [hunk 53]
```diff
@@ -14,10 +14,26 @@ def development_step4
...
+            </p>
+            <p>
+              You don't have to demonstrate strength in all of the areas below.
+            </p>
+            <p>
+              The guidance notes below each section are not exhaustive. Answer the questions in a way that best suits your organisation.
```
Comment (matthewford) [inline app/forms/qae_2014_forms/sustainable_development/sustainable_development_step4.rb]: Line is too long. [136/100]

Comment (hammerfunk) [inline app/forms/qae_2014_forms/sustainable_development/sustainable_development_step4.rb]: Ignore

File: app/forms/qae_2014_forms/sustainable_development/sustainable_development_step4.rb (hunk 52 above)
Comment (hammerfunk) [inline app/forms/qae_2014_forms/sustainable_development/sustainable_development_step4.rb]: Ignore

**Comment by RuslanHamidullin** [comment]
Good job @hammerfunk  :+1:

**Pull request merged**